### ISSUE TITLE
Block specific code actions

### ISSUE THREAD

**Issue by wstevick** [kosayoda/nvim-lightbulb#58]
Can there be a way to block code actions that match a given pattern. My use case is that denols provides the action "Move to a new file" for *all* lines of a file. It would be very helpful to be able to say something like
```json
{
    "ignore_actions": {
        "javascript": [
            "Move to a new file"
        ]
    }
}
```

Or whatever format.

**Comment by wookayin** [comment]
Using a filter **function** or predicate that takes `lsp.CodeAction` or `lsp.Command` lua table (and returns boolean whether to ignore or not) would be a much better and general solution.

**Comment by kosayoda** [comment]
Hello! The `filter` feature is now implemented in [18c54dd](https://github.com/kosayoda/nvim-lightbulb/commit/18c54dd28d25b5bad7401f27fb73224f9ccac61a).

Thank you very much for your patience.
